arch/x86/kernel/cpu/resctrl/rdtgroup.c

```diff
@@ -137,11 +137,20 @@ static void closid_init(void)
 
 static int closid_alloc(void)
 {
-	u32 closid = ffs(closid_free_map);
+	u32 closid;
+	int err;
```

Comment:
this one differs with upstream instead of err they are using cleanest_closid.

Reply:
@clsotog I have cleaned up.
the base kernel is [0].

[0], https://github.com/NVIDIA-BaseOS-6/linux-nvidia-6.5/tree/MPAM-dev